packages/onboarding/src/step-container-v2/components/ContentWrapper/style.scss

```diff
@@ -6,19 +9,31 @@
 	width: 100%;
 	box-sizing: border-box;
 	flex: 1;
-	max-width: 1038px;
 
 	&.padding {
 		padding: var( --step-container-v2-content-block-padding )
 			var( --step-container-v2-content-inline-padding );
 	}
 
 	&.wide {
-		max-width: 1440px;
+		--step-container-v2-content-max-width: calc(
+			#{$break-wide} + 2 * var( --step-container-v2-content-inline-padding )
+		);
+		max-width: var( --step-container-v2-content-max-width );
 	}
 
-	&.full {
-		max-width: 100%;
+	&.huge {
+		--step-container-v2-content-max-width: calc(
+			#{$break-huge} + 2 * var( --step-container-v2-content-inline-padding )
+		);
+		max-width: var( --step-container-v2-content-max-width );
+	}
+
+	&.xhuge {
+		--step-container-v2-content-max-width: calc(
+			#{$break-xhuge} + 2 * var( --step-container-v2-content-inline-padding )
+		);
+		max-width: var( --step-container-v2-content-max-width );
```

Comment:
Does the `max-width` rules have to be defined multiple times? Or should it be moved up a level. I'm not a CSS variable expert.